res/themes/light/css/_mods.scss

```diff
@@ -1,30 +1,3 @@
-// sidebar blurred avatar background
-//
-// if backdrop-filter is supported,
-// set the user avatar (if any) as a background so
-// it can be blurred by the tag panel and room list
-
-@supports (backdrop-filter: none) {
-    .mx_LeftPanel {
-        background-image: var(--avatar-url, unset);
```

Comment:
Can get rid of where this var is set too https://github.com/matrix-org/matrix-react-sdk/pull/6054/files#diff-bb5eeeda9a8d5f7b2a4cbcce491cfbf482b2c704207b28a167d31992851e7c1aL128-L143